xls/ir/caret.cc

```diff
@@ -95,7 +95,9 @@ std::string PrintCaret(
     int64_t slack =
         terminal_width - (line_contents_owned.size() + line_number_width + 3);
     int64_t size =
-        std::max(0l, static_cast<int64_t>(line_contents_owned.size()) + slack);
+        std::max(
+          static_cast<int64_t>(0),
```

Comment:
What's the difference between `0l` and `static_cast<int64_t>(0)`?

Reply:
The exact definition of `long` is architecture-specific; for example, in ILP64, `int`s, `long`s, and pointers are 64 bits wide, whereas in LLP64, a `long` is 32 bits, whereas `long long`s and pointers are 64 bits. IIRC, OSX is LP64, which _should_ be fine here, but either A) I'm wrong, or B) the compiler is very particular.